db/seeds.rb

```
@@ -1,49 +1,133 @@
   email: "agent@example.com",
   password: "password",
   role: :agent,
-  organization: organization
+  department: "Support",
+  position: "Support Agent",
+  organization: organization,
+  team: support_team # Assign to Support Team
 )
 
-User.create!(
+viewer_user = User.create!(
```

rubocop (1.69.1)

[warning] Useless assignment to variable - `viewer_user`.

(Lint/UselessAssignment)